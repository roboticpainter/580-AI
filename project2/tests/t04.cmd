train_input2.txt train_output2.txt test_input2.txt test_output2.txt structure3.txt weights3.txt 100 
