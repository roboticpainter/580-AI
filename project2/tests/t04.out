0.050000000181 0.099999996081 0.070000003719 -0.080000001836 -0.039999998762 -0.059999995747 -0.080000000597 0.089999997591 0.060000001601 -0.050000001066 0.040000001583 0.039999999323 0.049999998576 -0.050000000830 -0.040000001290 0.089999997867 0.029999998059 0.039999999628 0.040000000480 0.059999999176 -0.050000001397 -0.059999998697 -0.039999998646 -0.059999996100 -0.040000001506 -0.079999999658 0.080000000007 0.030000001633 -0.049999996075 0.039999998353 
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
0.350000000000
